assets/css/style.css

```diff
@@ -356,10 +356,14 @@ code {
     font-size: 0.85em;
 }
 
-:not(pre) code {
+code {
     color: brown;
 }
 
 pre {
     overflow: scroll;
 }
+
+pre code {
+    color: white;
```

Comment:
not 100% on this approach.

1. I've tried a code block with no language defined. In this case, the background is white but the foreground is also white, so, it's a no-show.

2. If the intent is to have a nicer color scheme for code, I totally agree we could do a lot better.
  Hugo offers styles for syntax highlighting, and I have experience using them.
  There are some decent looking styles. https://gohugo.io/content-management/syntax-highlighting/